test/JDBC/expected/BABEL-4279.out

```diff
@@ -235,3 +245,12 @@ GO
 DROP TABLE t3
 GO
 
+-- psql
+ALTER SYSTEM SET babelfishpg_tsql.migration_mode = 'single-db';
+SELECT pg_reload_conf();
+GO
+~~START~~
+bool
+t
+~~END~~
```

Comment:
we should refrain from doing such stuff in the test file. What is the purpose of doing this?

Reply:
to set migration mode back to single-db.

Reply:
We should do that like this : 
```
INSERT INTO #temptable SELECT current_setting('babelfishpg_tsql.migration_mode')
```

then after all the test done 

```
DECLARE @mig_mode VARCHAR(10)
SET @mig_mode = (SELECT mig_mode FROM #temptable WHERE id_num = 1)
SELECT CASE WHEN (SELECT set_config('babelfishpg_tsql.migration_mode', @mig_mode, false)) IS NOT NULL THEN 1 ELSE 0 END
GO
```

Because we can't make an assumption the default is always single-mode

Reply:
make sense, updated.